[Feat] 저장 경로 변경

AstroNaut000 opened the pull request:
utils/hotel_utils 파일 내 load_model()의 model_uri 변경 및 위치 변환
print() 삭제

lazychoi reviewed (approved):
LGTM

pastnfast reviewed (approved):
LGTM

pastnfast reviewed (changes requested):
[feat-16 : comflict rebase] -> [feat-16 : conflict rebase]

AstroNaut000 committed:
feat-16: 저장 경로 수정

같은 모델에 대한 새 버전이 더 평가가 좋아
새 모델로 production이 될 경우 자동으로
production이 된 모델로 추적할 수 있게
저장 경로 지정을 변경

Issues: #16

Update hotel_router.py

AstroNaut000 committed:
Update hotel_utils.py

AstroNaut000 commented:
> 
바꿨습니다.

pastnfast reviewed (approved):
LGTM
